ui/DesignSystem/Component/RemoteHelperHint.svelte

```diff
@@ -0,0 +1,53 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+
+  import { Icon } from "../../DesignSystem/Primitive";
+  import Copyable from "./Copyable.svelte";
+
+  const dispatch = createEventDispatcher();
+
+  export let style = null;
+  export let hidden = false;
+</script>
+
+<style>
+  .info {
+    margin-top: 1rem;
+    background-color: var(--color-foreground-level-1);
+    border-radius: 4px;
+    padding: 0.5rem;
+    align-items: left;
+    text-align: left;
+  }
+
+  .close-hint-button {
+    float: right;
+    cursor: pointer;
+  }
+</style>
+
+<div class="info" {hidden} {style} data-cy="remote-helper-hint">
+  <div
+    data-cy="close-hint-button"
+    class="close-hint-button"
+    on:click={() => {
+      dispatch('hide');
+      hidden = true;
+    }}>
+    <Icon.Cross variant="small" />
+  </div>
+  <p style="margin-bottom: 0.75rem; color: display: flex;">
+    Before you get started, you need to add this to your shell configuration.
+    Not sure how?
+    <a
+      style="color: var(--color-foreground-level-6);"
+      class="typo-link"
+      href="https://radicle.xyz/#upstream-faq">
+      Read more
+    </a>
+    .
+  </p>
+  <Copyable>
+    <p class="typo-text-small-mono">export PATH="~/.radicle/bin:$PATH"</p>
```

Comment:
Are we planning to only make it easy for users of `bash` and `zsh`? Or could we have a toggle for different shells with the line to copy chanigng accordingly?

Reply:
I think we should keep it simple initially and add different options later on, if there is demand for it.

Reply:
Personally I use `fish`. :>

Jokes aside, it's not worth the effort, full steam ahead.

Reply:
> Personally I use fish. :>

I know :)

><(((º>